### PR TITLE
collections menu: change more dropdown spacing

### PR DESCRIPTION
**Pull request by MJedr**
Ref: #1115

### CODE REVIEW

File: ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss [hunk 1]
```diff
@@ -35,25 +35,35 @@
 
 .more-collections-menu {
   .ant-dropdown-menu {
+    position: absolute;
```
Comment (harunurhan) [inline ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss:38]: why is this needed?, position seems already OK.

File: ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss [hunk 2]
```diff
@@ -35,25 +35,35 @@
...
     background-color: $layout-header-background !important;
     min-width: 160px;
   }
 
   .dropdown-menu-item {
-    padding-top: 0.3em;
-    paddig-bottom: 0.3em;
+    height: 2.5rem;
```
[the remark this answers is not in the document]
Comment (MJedr) [reply ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss:44]: Yes, because Stella asked for the same spacing as in other dropdown's (and without it it is quite small)

File: ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss [hunk 3]
```diff
@@ -40,20 +40,28 @@
...
+    line-height: 2.5rem;
+    margin-top: 0.25rem;
+    padding: 0 1rem;
     &:hover {
       background-color: $layout-header-background !important;
     }
   }
 
+  .ant-dropdown-menu-item > a {
+    display: unset;
```
Comment (harunurhan) [inline ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss:52]: why is that necessary?

Comment (MJedr) [reply ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss:52]: Because without ```display: unset``` it has ```display: block``` by default and the underline is much lower than the link

**Review by harunurhan — APPROVED** [review]
